README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
It doesn't look like *all* source code files actually get indexed. Just those with file extensions that match a list you have. Should this documentation be adjusted then?
Why is it that native projects need to index more than what the PDB claims has been included in the binary?